frontend/test/posts-page.spec.ts

```diff
@@ -15,3 +15,20 @@ describe('Posts timeline page', () => {
 		expect(body).toContain("I'll be House Speaker")
 	})
 })
+
+describe('Toot details', () => {
+	it('should show an individual toot', async () => {
+		// Find a specific toot in the list
+		const exploreResponse = await fetch('http://0.0.0.0:6868/explore/')
+		const exploreBody = await exploreResponse.text()
+		const match = exploreBody.match(/href="\/(@georgetakei\/[0-9a-z-]*)"/)
+		if (!match || !match[1]) fail()
+
+		// Fetch the page for it and validate the result
+		const tootPath = match[1]
```

Comment:
Happy to change it to an expect.

`> generate more generic/difficult to diagnose` I actually don't think thats the case. With the proposed changes, the error you get is this:
![image](https://user-images.githubusercontent.com/204386/211646093-96d2ddd6-9a82-4115-b6fc-810258ddf6f4.png)

as opposed to something failing on the exact line where the issues actually is (here, it's the call to `.match`). I don't think `fail` is better, they're just both equally opaque.

Reply:
I am not sure, I guess it can be a personal preference thing

for me it is clear that you are looking for an element and you make sure/expect that it exists

if you prefer the fail I won't insist 🙂 

By the way, I just tried to use `fail` and it is failing the test because the `fail` method is not defined (not because the method makes the test fail), does that happen on your end as well?
![Screenshot 2023-01-10 at 20 36 14](https://user-images.githubusercontent.com/61631103/211657296-929f91ef-5538-4efd-87c7-4b66e1888e2a.png)